trin-history/src/jsonrpc.rs

```diff
@@ -171,7 +171,7 @@ impl HistoryRequestHandler {
                     let _ = request.resp.send(response);
                 }
                 HistoryEndpoint::RecursiveFindNodes(node_id) => {
-                    let node_id = discv5::enr::NodeId::from(node_id.0);
+                    let node_id = discv5::enr::NodeId::from(node_id.0.raw());
```

Comment:
I think we can remove this conversion and use `node_id.into()` on the next.

Reply:
solved